.idea/codeStyleSettings.xml

```diff
@@ -135,9 +138,8 @@
           <option name="PARENT_SETTINGS_INSTALLED" value="true" />
           <indentOptions>
             <option name="INDENT_SIZE" value="2" />
-            <option name="CONTINUATION_INDENT_SIZE" value="2" />
+            <option name="CONTINUATION_INDENT_SIZE" value="4" />
```

Comment:
@dkincade has been wrong before :-D

This file should be the exact copy of https://github.com/pentaho/pentaho-coding-standards/blob/master/codeStyleSettings.xml ... or better yet should not be included at all in this project.